exercises/pig-latin/src/test/java/PigLatinTranslatorTest.java

```diff
@@ -1,85 +1,157 @@
-    public static Collection<Object[]> data() {
-        return Arrays.asList(new Object[][]{
-                // Ay is added to words that start with vowels
-                {"apple", "appleay"},
-                {"ear", "earay"},
-                {"igloo", "iglooay"},
-                {"object", "objectay"},
-                {"under", "underay"},
+    @Test
+    public void testWordBeginningWithA() {
+        assertEquals("appleay", pigLatinTranslator.translate("apple"));
+    }
 
-                // Word beginning with vowel follwed by a 'qu'
-                {"equal", "equalay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithE() {
+        assertEquals("earay", pigLatinTranslator.translate("ear"));
+    }
 
-                // First letter and ay are moved to the end of words that start with consonants
-                {"pig", "igpay"},
-                {"koala", "oalakay"},
-                {"xenon", "enonxay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithI() {
+        assertEquals("iglooay", pigLatinTranslator.translate("igloo"));
+    }
 
-                // Word beginning with 'q' without a following 'u'
-                {"qat", "atqay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithO() {
+        assertEquals("objectay", pigLatinTranslator.translate("object"));
+    }
 
-                // Ch is treated like a single consonant
-                {"chair", "airchay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithU() {
+        assertEquals("underay", pigLatinTranslator.translate("under"));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithVowelAndFollowedByQu() {
+        assertEquals("equalay", pigLatinTranslator.translate("equal"));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithP() {
+        assertEquals("igpay", pigLatinTranslator.translate("pig"));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithK() {
+        assertEquals("oalakay", pigLatinTranslator.translate("koala"));
+    }
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithX() {
+        assertEquals("enonxay", pigLatinTranslator.translate("xenon"));
+    }
 
-                // Qu is treated like a single consonant
-                {"queen", "eenquay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithQWithoutAFollowingU() {
+        assertEquals("atqay", pigLatinTranslator.translate("qat"));
+    }
 
-                // Qu and a single preceding consonant are treated like a single consonant
-                {"square", "aresquay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithCh() {
+        // Ch is treated like a single consonant
```

Comment:
Instead of needed this comment you could try and fit the same information in the test title, e.g. ` testChTreatedLikeAConsonantAtTheBeginningOfAWord`. The test title will show up in the error message so it's useful if it describes what the test is testing as accurately as possible 🙂 

The same applies for the comments in the other tests. The comments were needed when the tests were parameterized since we didn't have individual test names. But now that we do I don't think we need the comments anymore. What do you think? 🙂

Reply:
Yeah that's reasonable, will make change accordingly